usdigitalresponse/usdr-gost
# Add ECS health checks for API tasks

### Ticket #3622
## Description

This PR provides configuration for Amazon ECS to check whether a running API task is healthy.

## Screenshots / Demo Video

## Testing

1. Start the API server (e.g. `yarn start:server`)
2. Run the following command:
    ```cli
    node packages/server/src/scripts/healthcheck.js
    ```
    By default, this will trigger a GET request to `http://localhost:3000/api/health` with a 5-second timeout, although you can override this behavior by providing a URL and timeout as positional arguments, e.g. to target `http://my-local-server:3001/api/health` with a 10-second timout:
    ```cli
    node packages/server/src/scripts/healthcheck.js http://my-local-server:3001/api/health 10000
    ```
    You should observe that this command prints the status code and exits successfully (exit code 0).
3. Stop the running API server.
4. Run the same command. Observe that it waits the configured number of seconds (default 5), then prints an error (presumably `ECONNREFUSED`), and exits with a failure (exit code 1).

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
